lls/cache.c

```diff
@@ -52,7 +52,27 @@ lls_cache_dump(struct lls_cache *cache)
 	LLS_LOG(DEBUG, "LLS cache (%s)\n=====================\n", cache->name);
 	index = rte_hash_iterate(cache->hash, &key, &data, &iter);
 	while (index >= 0) {
-		cache->print_record(cache, &cache->records[index]);
+		struct lls_record *record = &cache->records[index];
+		struct lls_map *map = &record->map;
+		char ip_str[MAX_INET_ADDRSTRLEN];
+		int ret = convert_ip_to_str(&map->addr, ip_str,
+			sizeof(ip_str));
+		if (ret < 0)
+			goto next;
```

Comment:
1. We can add `unlikely()` to the test;
2. An error message should be reported before `goto`ing.